docs/source/hpxml_to_openstudio.rst

```diff
@@ -517,7 +517,7 @@ Depending on the type of water heater specified, additional elements are require
 ========================================  ===================================  ===========  ==========  ===============  ========================  =================  =================  =========================================
 WaterHeaterType                           UniformEnergyFactor or EnergyFactor  FuelType     TankVolume  HeatingCapacity  RecoveryEfficiency        RelatedHVACSystem  UsesDesuperheater  WaterHeaterInsulation/Jacket/JacketRValue
 ========================================  ===================================  ===========  ==========  ===============  ========================  =================  =================  =========================================
-storage water heater                      required                             <any>        required    <optional>       required if non-electric                     <optional>         <optional>
+storage water heater                      required                             <any>        <optional>  <optional>       required if non-electric                     <optional>         <optional>
```

Comment:
For TankVolume and HeatingCapacity defaulting, you can point to Table 7 in the [Building America House Simulation Protocols](https://www1.eere.energy.gov/buildings/publications/pdfs/building_america/house_simulation.pdf). I believe it should be identical to the equations, but please verify.